tests/zfs-tests/tests/functional/events/events_common.kshlib

```diff
@@ -23,10 +23,35 @@
 # Copyright (c) 2017 by Lawrence Livermore National Security, LLC.
 # Use is subject to license terms.
 #
+# Copyright (c) 2020 by Delphix. All rights reserved.
+#
 
 . $STF_SUITE/include/libtest.shlib
 . $STF_SUITE/tests/functional/events/events.cfg
 
+#
+# wait for 'event' to show up in the log 'file'
+function file_wait_event # file event timeout
+{
+	file=$1
```

Comment:
This seems to be unused. Perhaps have it default to $ZED_DEBUG_LOG?

Reply:
There are still some external callers to `file_wait` that can possibly be converted to `file_wait_event` so I left the file input available.

Reply:
I think having an unused variable invites confusion. Even in my initial comment I didn't clearly understand what its purpose was intended to be.